src/wordrace.js

```diff
@@ -1,28 +1,51 @@
 const axios = require('axios');
 
 let correctWord = undefined;
-let isAnswered = false;
-let isGameRunning = false;
 let numberOfRounds = 0;
-let numberOfSeconds = 0;
+let isFirstReply = true;
+let isGameOngoing = false;
+let discordChannel = null;
 
-const handleWordRace = (interaction) => {
+const handleWordRace = async (interaction) => {
     if(interaction.commandName === 'wordrace'){
-        numberOfRounds = interaction.options.get('rounds');
-        numberOfSeconds = interaction.options.get('time');
-        getWord(interaction);
+        if(isGameOngoing){
+            interaction.reply('WAIT! A game is going on right now!');
```

Comment:
Can you specify which chat the game is ongoing here?

Reply:
sure thing, added a new commit that includes the chat in which the game is being played